File: web3rpc/rpc-specs/code-samples/python/klay/configuration/getParams.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockNumber = 89

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_params(blockNumber)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/configuration/getParams.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const blockNumber = 89;

  provider.klay
    .getParams(blockNumber, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
